mender-convert-package

```diff
@@ -239,6 +256,7 @@ if [ "${MENDER_COPY_BOOT_GAP}" == "y" ]; then
 fi
 
 # Burn Partitions
+log_info "Writing boot, root, and data partitions"
```

Comment:
This change does not seem to be related, could you put this in a separate commit?

Additionally it would make sense to break this up, e.g

```
log_info "Writing boot partition image"
disk_write_at_offset "${boot_part}" "${img_path}" "${boot_part_start}"
log_info "Writing rootfsa partition image"
disk_write_at_offset "${output_dir}/rootfs.img" "${img_path}" "${rootfsa_start}"
....
```

Reply:
Yes I could make a different commit, it just seemed very minor.  While watching the output, if you're not writing the "boot gap" then the last log output you see is writing the partition table, so you get the feeling it's taking way too long without these extra statements.

Reply:
The change makes sense, just put it a different commit to keep the logically separated. You can have the commit in the same PR.